resotoworker/resotoworker/cleanup.py

```diff
@@ -15,9 +15,7 @@ def cleanup():
     if ArgumentParser.args.collector and len(ArgumentParser.args.collector) > 0:
         clouds = '["' + '", "'.join(ArgumentParser.args.collector) + '"]'
         query_filter = f"and /ancestors.cloud.reported.id in {clouds} "
-    query = (
-        f"desired.clean == true and metadata.cleaned == false {query_filter}<-[0:]->"
-    )
+    query = f"desired.clean == true and metadata.cleaned == false {query_filter}<-default,delete[0:]->"
```

Comment:
```suggestion
    query = f"/desired.clean == true and /metadata.cleaned != true and /metadata.protected!=true {query_filter}<-default,delete[0:]->"
```
Not 100% sure where this query is posted - but since you use `/ancestors` maybe the other props also need the absolute path?